### DISCUSSION TITLE
編集リクエスト時のプレビュー画面同時表示時に画面スクロールが同期しない

### DISCUSSION THREAD

**Discussion by shiracamus** [increments/qiita-discussions#248] [要望・アイデア・不具合]
編集リクエスト時の画面でプレビュー同時表示モードにしたとき、記事入力欄をスクロールしてもプレビュー画面はスクロールせずに止まったままです。
記事投稿時や質問回答時では同期してスクロールします。

**Comment by ohakutsu** [comment]
@shiracamus 

お問い合わせいただきありがとうございます！

こちら、[先日リリースされたエディタのアップデート](https://blog.qiita.com/release-qiita-editor-beta-2022/)の「編集エリアとプレビューのスクロール位置の同期」がうまく動いていないということでお間違い無いでしょうか？

編集リクエスト時のスクロール位置の同期を僕の方でも確認をしてみたのですが、再現ができませんでした 💦 
引き続き調査をしてみますが、もしまだスクロール位置の同期がうまく動いていなければ、対象の記事を教えていただけませんでしょうか 🙏

**Reply by shiracamus** [reply]
私の環境がちょっと特殊で、Biscuit https://eatbiscuit.com/ja というマルチWebブラウザを使って Qiita 記事を読んでいます。
Qiita記事の三点メニューから「編集リクエストを送る」を右クリックして「新しいタブで開く」を選ぶと、編集画面だけが表示されてプレビュー画面が表示されません。その状態で 本アイコン をクリックしてプレビュー画面を右半分に表示すると同期スクロールしないのです。
画面をよく見ると上部にある「同時スクロール」チェックボックスにマークが付いておらず、このマークにチェックをつけることで同期スクロールするようになりました。
特殊環境ということで本件はクローズしてください。
お騒がせいたしました。

**Reply by ohakutsu** [reply]
@shiracamus 

そうだったのですね 💦 
スクロール位置の同期が動いているようでよかったです！

もし、また不具合かも？という現象があればご気軽にDiscussionよりお知らせいただけると幸いです！
